src/components/ProductRow.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
```

Comment:
Good work, here's an alternative solution for `ProductRow` (just as a different perspective/approach, you don't need to implement it).
```
function ProductRow(props) {
  const { stocked, name, price } = props.item;
 
  return (
    <tr style={ stocked ? {color: 'black'}: {color: 'red'}}>
      <td>{name}</td>
      <td>{price}</td>
    </tr>
  );
}